### PR TITLE
[OclRuntime] Added gpuMemCopy() function

### PR DESCRIPTION
**Pull request by AndreyPavlenko**

### CODE REVIEW

**Review by dchigarev — APPROVED** [review]
LGTM, thank you!

File: lib/gc/ExecutionEngine/OpenCLRuntime/OpenCLRuntimeWrappers.cpp [hunk 1]
```diff
@@ -390,6 +390,14 @@ extern "C" OCL_RUNTIME_EXPORT void gpuMemFree(GPUCLQUEUE *queue, void *ptr) {
   }
 }
 
+extern "C" OCL_RUNTIME_EXPORT void gpuMemCopy(GPUCLQUEUE *queue, void *dst,
+                                              void *src, uint64_t size) {
+  auto func = queue->ext_table_ ? queue->ext_table_->enqueneMemcpy
```
Comment (Menooker) [inline lib/gc/ExecutionEngine/OpenCLRuntime/OpenCLRuntimeWrappers.cpp:395]: Sorry, it is a typo of mine: `enquene` should be `enqueue`. Shall we fix it by the way?

**Comment by Menooker** [comment]
LGTM

**Review by Menooker — APPROVED** [review]

File: lib/gc/ExecutionEngine/OpenCLRuntime/OpenCLRuntimeWrappers.cpp (hunk 1 above)
Comment (AndreyPavlenko) [reply lib/gc/ExecutionEngine/OpenCLRuntime/OpenCLRuntimeWrappers.cpp:395]: Didn't notice this, thanks. I'll add a fix in this PR.

**Pull request merged**